.sync.yml

```diff
@@ -2,11 +2,15 @@
   componentName: crossplane
   feature_goldenTests: true
   testMatrix:
-    key: provider
+    key: instance
     entries:
       - defaults
       - openshift4
 
 docs/antora.yml:
   name: crossplane
   title: crossplane
+
+.github/workflows/test.yaml:
+  makeTarget: test -e instance=${{ matrix.instance }}
+  goldenTest_makeTarget: golden-diff -e instance=${{ matrix.instance }}
```

Comment:
Once more, please move this fix to a separate PR (or commit it directly to master) and rerun modulesync to get the updated GitHub actions config.

Reply:
sure lets do that

Reply:
ModuleSync [PR](https://github.com/projectsyn/component-crossplane/pull/21)